web/build.gradle

```diff
@@ -36,19 +34,27 @@ tasks.named("test") {
 }
 bootRun {
     systemProperty("spring.profiles.active", "testability,dev")
+    systemProperty("application.dir", "${applicationDir}/devops/dev")
```

Comment:
We need to remove /devops/dev now when it is included in the applicationDir variable?